src/app/components/get-involved/get-involved.component.scss

```diff
@@ -69,10 +77,15 @@
 
 
 @include screen-medium {
+  .get-involved-container {
+    .get-involved-title {
+      margin: 0px;
+    }
+  }
   .get-involved-section {
     div {
       .get-involved-section-title {
-        margin-top: 50px;
+        margin: 0px !important;
```

Comment:
Was `!important` needed? (and in other places)

Reply:
@lunayach yeah actually it is not needed here and I've used `!important in one other place` also but there it is needed, please have a look

Reply:
Okay.